.github/workflows/eks.yaml

```diff
@@ -223,3 +223,12 @@ jobs:
           description: Connectivity test cancelled
           state: pending
           target_url: ${{ env.check_url }}
+
+      - name: Send slack notification
+        if: ${{ (cancelled() || failure()) && (github.event_name == 'schedule' || github.event_name == 'push') }}
```

Comment:
When is `github.event_name == 'push'` true? I thought this workflow was only triggered on PR comments or and schedule.

Reply:
I copy pasted the same step to all workflows regardless of whether the `push` was actually in the file or not.

Reply:
Ok, maybe it's easier/cleaner that way.